lib/views/question/question_container_view_model.dart

```diff
@@ -1,30 +1,89 @@
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:kayla_flutter_ic/model/enum/display_type.dart';
+import 'package:kayla_flutter_ic/model/survey_detail.dart';
+import 'package:kayla_flutter_ic/usecases/base/base_use_case.dart';
+import 'package:kayla_flutter_ic/usecases/survey/get_current_survey_detail_use_case.dart';
 import 'package:kayla_flutter_ic/utils/route_paths.dart';
+import 'package:kayla_flutter_ic/views/question/answers_container_ui_model.dart';
 import 'package:kayla_flutter_ic/views/question/question_container_state.dart';
 import 'package:kayla_flutter_ic/views/question/question_container_ui_model.dart';
 
 class QuestionContainerViewModel extends StateNotifier<QuestionContainerState> {
   String get surveyId => _surveyId ?? '';
   int get questionNumber => _questionNumber ?? 0;
+  SurveyDetail get survey => _survey ?? SurveyDetail.empty();
```

Comment:
usually, we will not define public getter if we don't create any outside references for the variables. I think you can use `_survey`, or `_surveyId` internally, right?